README.md

```diff
@@ -125,15 +125,14 @@ Read more about how to install the example webhook [here](deploy/kubernetes/webh
 
 * `--http-endpoint`: The TCP network address where the HTTP server for diagnostics, including metrics and leader election health check, will listen (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means the server is disabled.
 
-* `--metrics-address`: (deprecated) The TCP network address where the prometheus metrics endpoint will run (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint is disabled.
```

Comment:
Was this removed this release? I think we'll need to keep it until the next major.

Reply:
This is a mistake in the README.  "metrics-address" was never added to the snapshot controller.